test/blackbox-tests/test-cases/pkg/build-package-logs.t

```diff
@@ -0,0 +1,44 @@
+Test the error message when installing package that fails.
+
+  $ . ./helpers.sh
+  $ make_lockdir
+
+Make a project with two packages, one successful and, one that fails:
+
+  $ cat > dune-project << EOF
+  > (lang dune 3.12)
+  > EOF
+
+We create a package with a failing command that throws an error:
+
+  $ make_lockpkg x << EOF
+  > (version 0.0.1)
+  > (build 
+  >    (progn
+  >       (run cat i_dont_exist)))
+  > EOF
+
+Building the package should fail and print an error:
+
+  $ build_pkg x
+  File "dune.lock/x.pkg", line 4, characters 11-14:
+  4 |       (run cat i_dont_exist)))
+                 ^^^
+  Package x fails to build
+  /usr/bin/cat: i_dont_exist: No such file or directory
+  
+  [1]
+
+We create a package with a succeeding command that displays some text:
+
+  $ make_lockpkg y << EOF
+  > (version 0.0.1)
+  > (build 
+  >    (progn
+  >       (run echo "Success!")
+  >       (run true)))
+  > EOF
+
+Building the package should succeed and print no output:
+
+  $ build_pkg y
```

Comment:
It would be nice if you could make sure that the package `y` succeeded to install. I think `helpers.sh` has a function to display the "cookie" of a package and that should presumably exist when the package was successfully installed.

```sh
  $ show_pkg_cookie y
```

or something like this.